### ISSUE TITLE
Error: Failed to start Omnisharp

### ISSUE THREAD

**Issue by rashidalnaemi** [dotBunny/VSCode#96]
```
[INFO] Starting OmniSharp at 'c:\Users\rashid\Documents\UnityProjects\2DTesting\2DTesting.sln'...
[INFO] Started OmniSharp from 'C:\Users\rashid\.vscode\extensions\ms-vscode.csharp-0.3.7\bin\omnisharp.cmd' with process id 8928...
System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Framework.Runtime.Loader, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified.
File name: 'Microsoft.Framework.Runtime.Loader, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'
   at dnx.host.Bootstrapper.RunAsync(List`1 args)
   at dnx.host.RuntimeBootstrapper.ExecuteAsync(String[] args)
   at dnx.host.RuntimeBootstrapper.Execute(String[] args)

WRN: Assembly binding logging is turned OFF.
To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
Note: There is some performance penalty associated with assembly bind failure logging.
To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].

[ERROR] Error: Failed to start OmniSharp

```

VSCode 1.1.0
Installed plugin from Asset Store
Unity 5.3.4f1
Installed Latest Mono from their website.

Any ideas? I don't know how to even start debugging this.

**Comment by rashidalnaemi** [comment]
More Info:
https://github.com/OmniSharp/omnisharp-server/issues/213

Probably Omnisharp's fault. Keeping this here in case someone else needs help with it.

**Comment by corum** [comment]
Looks like Omnisharp is no longer included by default since 1.1.x version of VSCode
I solved this issue by installing a new extension, based on Omnisharp.
Do the trick by hitting F1 inside VSCode and typing >ext install c#
https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp
There are several changes that impact .NET Core SDK.
I suggest users to read carefully what reported in the extension page.
